notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        base_url = "https://datastudio.google.com/c/u/0/reporting"
</pre><p>Have you tested the ability for the public to access these templates?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
It is the same template and code used by the [forecasting demo notebook](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb), so I assumed it was tested.

Reply:
<p>I would double-check this. I had trouble with access to DataStudio when I went through the forecasting demo notebook. Refer to my friction log: https://docs.google.com/document/d/18pQgud8hFWFt0dkXUpH4e3shu5dtwsqKSdtTtz4Z9wk</p><p>Is there a way to visualize the forecasts within the notebook, without having to send the user to DataStudio?</p>

Reply:
<p>Also, the forecasts is shown with a continuous plot, but I think a discrete plot would make more sense. Can we change this?</p>